### PR TITLE
Create basic structure and translate home for Arabic

### PR DESCRIPTION
**Pull request by mboukhalfa**
This PR adds the minimal translation related to the file `/home/_index.md`

### CODE REVIEW

**Comment by adowair** [comment]
Thanks for re-opening this @mboukhalfa. I have a draft PR #44862  open for `data/i18n/ar/ar.toml` and your work here is vital—I will incorporate your changes here into it and credit you as co-author, in order to keep that unit of work independent.

File: assets/scss/_custom.scss [hunk 1]
```diff
@@ -154,6 +154,18 @@ body.td-404 main .error-details {
...
+  .navbar-brand {
+    left: 16px;
+  }
+  .announcement-main {
+
+    direction: ltr;
+    text-align: left;
+
+  }
+}
```
Comment (sftim) [inline assets/scss/_custom.scss:168]: - This change belongs in its own PR.
- Ideally, make it work for all LTR languages and not just Arabic.

SIG Docs can help you make and land that PR in _main_ so that the Arabic work can build upon it.

File: content/ar/docs/concepts/overview/_index.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+---
+reviewers:
+- mboukhalfa
+- oxPraGa
```
Comment (sftim) [inline content/ar/docs/concepts/overview/_index.md:4]: I'd omit page-specific reviewers; they don't add much (you can edit the whole team instead using `OWNERS` files).

File: i18n/ar.toml
Comment (sftim) [inline i18n/ar.toml:1]: This should be a symbolic link, not a file. SIG Docs can help you make a symbolic link if you are not sure how to.

File: hugo.toml
Comment (sftim) [inline hugo.toml:1]: If you can, avoid changing this file.

File: data/i18n/ar/ar.toml
Comment (sftim) [inline data/i18n/ar/ar.toml:1]: Please avoid including untranslated strings. You can mark what's omitted with a comment, but don't have English text where we eventually want to see Arabic.

File: content/ar/docs/home/supported-doc-versions.md
Comment (sftim) [inline content/ar/docs/home/supported-doc-versions.md:1]: This file is not in Arabic - let's not include it.

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -197,6 +197,10 @@ aliases:
     - Arhell
     - idvoretskyi
     - MaxymVlasov
+  sig-docs-ar-owners: # Admins for Arabic content
+    - mboukhalfa
+  sig-docs-ar-reviews: # PR reviews for Arabic content
+    - mboukhalfa
```
Comment (sftim) [inline OWNERS_ALIASES:203]: For now, omit this change.

**Review by sftim — COMMENTED** [review]
Thanks for this work. I have some feedback you'd need to account for before we can merge this, even to a branch you're not yet publishing to the live site

- avoid including any English files
- let's split the important CSS change out into its own pull request, so we can make sure we merge the right thing
- one of the localization strings files actually needs to be a symbolic link

/hold
I'd want to omit the change to `OWNERS_ALIASES` before removing the hold, and I'd want the PR to avoid including any files with substantial English text before we give this LGTM.

It's impressive, and we're here to help - with some more changes, this **can** land.

**Comment by sftim** [comment]
Even though there is more to do, I'd like to invite reviews by people who can speak both Arabic and English.

File: content/ar/OWNERS
Comment (sftim) [inline content/ar/OWNERS:1]: For now, omit this change (but we can help get it added once the team is up and running).

**Comment by seokho-son** [comment]
Hi @mboukhalfa,
As a lead in the localization subproject, I'm particularly eager to see a collaborative effort this time around, especially given the several previous attempts to initiate the Arabic version by individual contributors.

I suggest we use https://github.com/kubernetes/website/issues/44682 to rally active members for the Arabic localization. It's crucial to remember that localizing Kubernetes documents is a complex task due to the sheer volume of content. Maintaining these documents is likely more challenging than setting up the basic configuration for Arabic, and it's a task that cannot be sustained without a committed team of active contributors. (I hope to discuss with contributors who are eligible for reviewer/approver for Arabic content in #44682 . Please note that a candidate needs to be Kubernetes github organization member eventually)

With a focus on teamwork and guided steps, I'm hopeful that we will not lose our way and achieve a successful outcome together. 😉

**Comment by seokho-son** [comment]
cc: @a-mccarthy @natalisucks :)

File: content/ar/docs/home/supported-doc-versions.md
Comment (seokho-son) [reply content/ar/docs/home/supported-doc-versions.md:1]: - PTAL: https://github.com/kubernetes/website/pull/44698

**Comment by seokho-son** [comment]
Hi @mboukhalfa 
- Regarding this PR, I suggest separating some contents to simply and remove duplicated works. 
  - Let's remove files related with `(data) i18n/ar/*` i18n strings and utilize https://github.com/kubernetes/website/pull/44862 from @adowair 
  - Let's remove content/ar/docs/home/`supported-doc-versions.md` and utilize https://github.com/kubernetes/website/pull/44698
  
 (same comment here: https://github.com/kubernetes/website/issues/44682#issuecomment-1919099877)

File: assets/scss/_custom.scss (hunk 1 above)
Comment (sftim) [reply assets/scss/_custom.scss:168]: I'm happy to support the work to complete LTR support; I don't plan to work on it all myself.

Comment (sftim) [reply assets/scss/_custom.scss:168]: We should definitely separate it out so it can land as its own change, and into _main_.

**Comment by seokho-son** [comment]
Hi @mboukhalfa
Please give your feedback for the above review comment. I think many Arabic localization members are waiting for your response.
If you don't have enough time, let us know. :)

**Comment by mboukhalfa** [comment]
> Hi @mboukhalfa Please give your feedback for the above review comment. I think many Arabic localization members are waiting for your response. If you don't have enough time, let us know. :)

Thanks for the feedback I will address the comments ASAP and push

File: assets/scss/_custom.scss (hunk 1 above)
Comment (mboukhalfa) [reply assets/scss/_custom.scss:168]: I think should be fixed in https://github.com/google/docsy

**Comment by mboukhalfa** [comment]
Close as the content was split into file by file PRs mentioned above
/close
